src/Bicep.Core/TypeSystem/Az/OciTypeLoader.cs

```diff
@@ -7,22 +7,22 @@
 using System.IO.Compression;
```

Comment:
I notice the file is still under a TypeSystem/Az path, did you want to change that?

Reply:
Good catch, yes! I will change it :)